TMS.Net07.Homework.Calculator/Easy.cs

```diff
@@ -0,0 +1,125 @@
+                && (targetCurrency.ToLower() == "byn" || targetCurrency.ToLower() == "rub" || targetCurrency.ToLower() == "eur" || targetCurrency.ToLower() == "usd"))
+            {
+                Console.WriteLine("Input amount: ");
+                var input = Console.ReadLine();
+                if (decimal.TryParse(input, out var amount))
+                {
+                    CurrencyTranslation(sourceCurrency, targetCurrency, amount);
+                }
+                else
+                {
+                    Console.WriteLine("Wrong amount");
+                }
+            }
+            else
+            {
+                Console.WriteLine("Incorrect input");
+            }
+        }
+
+        static void CurrencyTranslation(string sourceCurrency, string targetCurrency, decimal amount)
+        {
+
+            if (sourceCurrency.ToLower() == "byn")
+            {
+                if (targetCurrency.ToLower() == "usd")
+                {
+                    Console.WriteLine($"{amount} {sourceCurrency} is equal to {amount * 0.38m} {targetCurrency}");
+                }
+                else if (targetCurrency.ToLower() == "eur")
+                {
+                    Console.WriteLine($"{amount} {sourceCurrency} is equal to {amount * 0.32m} {targetCurrency}");
+                }
+                else if (targetCurrency.ToLower() == "rub")
+                {
+                    Console.WriteLine($"{amount} {sourceCurrency} is equal to {amount * 28.80m} {targetCurrency}");
+                }
+                else if (targetCurrency.ToLower() == "byn")
+                {
+                    Console.WriteLine($"{amount} BYN is equal to {amount} BYN");
+                    return;
+                }
+            }
+            else if (sourceCurrency.ToLower() == "usd")
+            {
+                if (targetCurrency.ToLower() == "byn")
+                {
+                    Console.WriteLine($"{amount} {sourceCurrency} is equal to {amount * 2.63m} {targetCurrency}");
+                }
+                else if (targetCurrency.ToLower() == "rub")
+                {
+                    Console.WriteLine($"{amount} {sourceCurrency} is equal to {amount * 75.71m} {targetCurrency}");
+                }
+                else if (targetCurrency.ToLower() == "eur")
+                {
+                    Console.WriteLine($"{amount} {sourceCurrency} is equal to {amount * 0.83m} {targetCurrency}");
+                }
+                else if (targetCurrency.ToLower() == "usd")
+                {
+                    Console.WriteLine($"{amount} USD is equal to {amount} USD");
+                    return;
+                }
+            }
+            else if (sourceCurrency.ToLower() == "eur")
+            {
+                if (targetCurrency.ToLower() == "byn")
+                {
+                    decimal EURtoBYN = 3.16m;
```

Comment:
```suggestion
                    const decimal EURtoBYN = 3.16m;
```
Compiler substitutes constant values into places, where they are used, instead of their names. That's why they don't require additional memory while an application running time. And that's why it's better to use them when values are not supposed to be changed

Reply:
done